sdk/servicebus/azure-servicebus/azure/servicebus/_common/receiver_mixins.py

```diff
@@ -50,12 +50,11 @@ def _populate_attributes(self, **kwargs):
         self._last_received_sequenced_number = None
         self._message_iter = None
         self._connection = kwargs.get("connection")
-        prefetch_count = kwargs.get("prefetch_count", 0)
-        if int(prefetch_count) < 0 or int(prefetch_count) > 50000:
+        self._prefetch_count = int(kwargs.get("prefetch_count", 0))
+        if self._prefetch_count < 0 or self._prefetch_count > 50000:
             raise ValueError(
                 "prefetch_count must be an integer between 0 and 50000 inclusive."
             )
-        self._prefetch_count = prefetch_count + 1
```

Comment:
It seems like a reasonable direction to me that we would do this, and keep the value of `self._prefetch_count` actually at the value of "prefetch" rather than conflating it with `link_credit`. However I would be cautious of doing this at this cost of randomly adding `+ 1` all over the place - as it will be very tricky to keep track of how we should be handling this value..... so I think this is probably the right direction, but we might need to be careful as to how we're handling "prefetch" vs "link credit"

Reply:
makes sense! I think doing the + 1 in the Pyamqp/UamqpTransports when creating the ReceiveClient (with a comment) might be most clear?